lldb/packages/Python/lldbsuite/test/lldbutil.py

```diff
@@ -778,7 +784,11 @@ def run_to_breakpoint_do_run(test, target, bkpt, launch_info = None):
     threads = get_threads_stopped_at_breakpoint(
                 process, bkpt)
 
-    test.assertTrue(len(threads) == 1, "Expected 1 thread to stop at breakpoint, %d did."%(len(threads)))
+    buf = '\n'
+    for i in range(launch_info.GetNumEnvironmentEntries()):
+        buf += 'Env: ' + launch_info.GetEnvironmentEntryAtIndex(i) + '\n'
+
+    test.assertTrue(len(threads) == 1, "Expected 1 thread to stop at breakpoint, %d did. Launch info: %s"%(len(threads), buf))
```

Comment:
What is the output in case of failure here? [just curious :)]

Reply:
The old assertion message (expected 1 thread but found 0), followed by a newline-separated list of the environment variables forwarded to the inferior.